crates/polars-plan/src/dsl/function_expr/temporal.rs

```diff
@@ -78,6 +81,29 @@ pub(super) fn datetime(
     use polars_core::export::chrono::NaiveDate;
     use polars_core::utils::CustomIterTools;
 
+    if s.iter().any(|s| s.is_empty()) {
+        return Ok(Column::new_empty(
+            PlSmallStr::from_static("datetime"),
```

Comment:
Can we hoist this into a variable at the start of the function? Then we can reuse that here and on line 207.

Reply:
Sure. Updated in 8b485625e3bbdf063e4e88ca27156ff0ee3c4129.